### ISSUE TITLE
[ISSUE] TipJar extension does not generate QR-Code

### ISSUE THREAD

**Issue by DoktorShift** [lnbits/tipjar#8]
### Issue
When press "Submit" Button the TipJar extension does not throw out an QR-Code.

### Replicate the Bug
1. use Lnbits v0.11.2
2. TipJar v0.2.4
3. open extension
4. fill out the blanks
5. Press "Submit" to generate Invoice (QR-Code)

**Logs out of my LNBits instance**
![LNBits logs_tipJar](https://github.com/lnbits/tipjar/assets/106493492/46ed9987-479c-4644-b8ce-9f58d7b22534)

More Input

![h](https://github.com/lnbits/tipjar/assets/106493492/f058f398-340e-4910-815b-d12fcfa5025d)

Cause of the problems fetching BTC rate from EXIR i tryed at diffrent times with working internet connection.

**Comment by talvasconcelos** [comment]
Can't really reproduce the error. 

LNbits fetches fiat exchange rate from 8 different places, the fact that 1 fails doesn't make any issue!

**Comment by DoktorShift** [comment]
this issue is related to the bug reported in https://github.com/lnbits/tipjar/issues/9 and still existing in lnbits 0.12.1 with tipjar 0.2.4

**Comment by DaxSosa** [comment]
In my case I am using it through cloud flare on my node and it does not generate anything,

https://lnbits.daxsosa.com/tipjar/1

**Comment by dni** [comment]
your issue seems to be a internal routing one. 
```
"Client error '404 Not Found' for url 'http://0.0.0.0:3007/satspay/api/v1/charge'\nFor more information check: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404"
```

**Comment by DaxSosa** [comment]
> your issue seems to be a internal routing one.
> 
> ```
> "Client error '404 Not Found' for url 'http://0.0.0.0:3007/satspay/api/v1/charge'\nFor more information check: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404"
> ```

I already modified the output using cloudflare but it remains the same, it doesn't give me the qr, I am using it from an umbrel node and cloudflare

https://lnbits.daxsosa.com/tipjar/2

**Comment by talvasconcelos** [comment]
Fixed on Satspay, next release 0.3.2
